front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -511,30 +485,6 @@ const AdminProjectsProjectGeneral = () => {
             </StyledSectionField>
           )}
 
-          <StyledSectionField>
```

Comment:
This doesn't seem to have been used here - the component never rendered.

Reply:
My best guess is that we forgot to remove it when we were working on getting rid of continuous projects but I'm really not 100% sure.